[CI:DOCS] Document --userns=auto behaviour for rootless users (#20607)
containers/podman: docs/source/markdown/options/userns.container.md

**auto**[:_OPTIONS,..._]: automatically create a unique user namespace.

* `rootful mode`: The `--userns=auto` flag requires that the user name __containers__ be specified in the /etc/subuid and /etc/subgid files, with an unused range of subordinate user IDs that Podman containers are allowed to allocate.

  	   Example: `containers:2147483647:2147483648`.

> * `rootless mode`: The users range from the /etc/subuid and /etc/subgid files will be used. Note if you run a single container without using --userns=auto, it will use the entire range of UIDs and not allow further subdeviding. See subuid(5).

Reviewer: ```suggestion
* `rootless mode`: The users range from the /etc/subuid and /etc/subgid files will be used. Note running a single container without using --userns=auto will use the entire range of UIDs and not allow further subdividing. See subuid(5).
```
